Add feature extraction module (#8)
WayScience/mitocheck_data: 3.extract_features/README.md

> # 3. Extract Features

Reviewer: Somewhere in this README (or maybe in a separate issue), can you describe what you changed in the DeepProfiler config file that is different from the LUAD config?

Author: Done in https://github.com/WayScience/mitocheck_data/issues/10. Will close this issue after attaching the new config file used for mitosis movie feature extraction.

In this module, we present our pipeline for extracting features from the mitosis movies.

> 

Reviewer: ```suggestion

```

Delete superfluous extra new line

### Feature Extraction

We use [DeepProfiler](https://github.com/cytomining/DeepProfiler), commit [`2fb3ed3`](https://github.com/cytomining/DeepProfiler/commit/2fb3ed3027cded6676b7e409687322ef67491ec7), to extract features from the mitosis movies. 

We use a [pretrained model](https://github.com/broadinstitute/luad-cell-painting/tree/main/outputs/efn_pretrained/checkpoint) from the [LUAD Cell Painting repository](https://github.com/broadinstitute/luad-cell-painting) with DeepProfiler.
[Caicedo et al., 2022](https://www.molbiolcell.org/doi/10.1091/mbc.E21-11-0538) trained this model to extract features from Cell Painting data.
This model extracts features from the DNA (nuclei) channel and is thus a good selection for our use case.

## Step 1: Setup Feature Extraction Environment

### Step 1a: Create Feature Extraction Environment

```sh
# Run this command to create the conda environment for Segmentation data
conda env create -f 2.feature_extraction_env.yml
```

> ### Step 1b: Activate Feature Extraction  Environment

Reviewer: ```suggestion
### Step 1b: Activate Feature Extraction Environment

```

```sh

> # Run this command to activate the conda environment for Segmentation data

Reviewer: ```suggestion
# Run this command to activate the conda environment for Deep Profiler feature extraction

```

conda activate 3.feature_extraction_mitocheck
```

## Step 2: Install DeepProfiler

> ### Step 2a: Clone Resository

Reviewer: ```suggestion
### Step 2a: Clone Repository

```

Clone the DeepProfiler repository into 3.extract_features/ with 

> ```console

Reviewer: ```suggestion
```console
# Make sure you are located in 3.extract_features/
cd 3.extract_features/
```

Author: Fixed in https://github.com/WayScience/mitocheck_data/pull/8/commits/7ac793c5f893c7a57a56139c8e7423a0aa32fa8b.

git clone https://github.com/cytomining/DeepProfiler.git
```

> ### Step 2b: Install Resository

Reviewer: ```suggestion
### Step 2b: Install Resository

```
